frontend/modelarchive/src/test/java/com/amazonaws/ml/mms/archive/ModelArchiveTest.java

```diff
@@ -25,10 +25,10 @@
 
     @BeforeTest
     public void afterTest() {
-        output = new File("build/tmp/test/noop.model");
+        output = new File("build/tmp/test/noop.mar");
```

Comment:
Should we have both .model and .mar in the tests? For backwards compatibility. 0.1 wit .model and 1.0 with .mar